lib/determinator/cache/fetch_wrapper.rb

```diff
@@ -1,20 +1,49 @@
 module Determinator
   module Cache
     class FetchWrapper
-      # @param cache [#fetch] An instance of a cache class which implements #fetch like ActiveSupport::Cache does
-      def initialize(cache)
-        @cache = cache
+      # @param *caches [#fetch] n instances of an ActiveSupport::Cache (or eqivelent)
```

Comment:
You may want to explain that the cache at the left of the array will be used/checked before the ones on the right :) Also `s/eqivelent/equivalent` :)